### DISCUSSION TITLE
How to keep layer under text smooth?

### DISCUSSION THREAD

**Discussion by IanVS** [OrcaSlicer/OrcaSlicer#5666] [Q&A]
I'm printing a design that has text on top of a flat surface.  I want the flat surface to be a plain, monotonic surface, and the text to go straight onto it, but instead, the slicer adds small artifacts on the layer under the text.  Is there any way to avoid that?  It kind of makes a mess, especially with silk filament.

<img width="866" alt="image" src="https://github.com/SoftFever/OrcaSlicer/assets/4616705/9c2525be-8340-46b7-aea6-e947fdf04d8d">
<img width="852" alt="image" src="https://github.com/SoftFever/OrcaSlicer/assets/4616705/86b04cb0-a873-4f11-9c45-1f62a5b0bb1d">

**Comment by Avpman2** [comment]
+1
